docs/design/disabling_elements.rst

```diff
@@ -0,0 +1,55 @@
+Disabling interface elements
```

Comment:
```suggestion
Making interface elements inactive or unavailable
```

Let's not use disabling and instead talk about what we're actually doing - making something inactive or unavailable.

Reply:
We're doing this: making users become not allowed to interact with that thing, effectively removing their ability of taking actions

in the code, we use:
`cursor: not-allowed;`
to show the red X cursor as a feedback

another way to say is that we're removing pointer events, as this attribute is also appended to ensure users have their most deep desire and wishes to interact completely blocked:
`pointer-events: none;`

how would you find better to call?

Reply:
it's great to pick 1 word to avoid having the synonyms (inactive or unavailable) there in every use